base.js

```diff
@@ -0,0 +1,141 @@
+module.exports = {
+  plugins: [
+    // 未使用import文の自動削除のために利用
+    "unused-imports",
+  ],
+  extends: [
+    "airbnb-base",
+    // "airbnb-typescript/base", // TODO(hori-ryota): 未対応なのでいったんコメントアウト。入れたい。
+    "plugin:@typescript-eslint/recommended",
+    "plugin:eslint-comments/recommended",
+    // NOTE(hori-ryota): prettierは全設定の最後に記述する必要があるため、ここでは設定せずextendsする側でそれぞれ設定する
+  ],
+  rules: {
+    /**
+     * eslint
+     */
+    "arrow-body-style": "off",
+    camelcase: [
+      "error",
+      {
+        properties: "never",
+        ignoreDestructuring: false,
+        allow: ["^[A-Z].*_[A-Z].*"],
+      },
+    ],
+    "default-case-last": "error", // 統一したいだけなのでfirstでもいい
+    "default-param-last": "off", // optional paramを最後尾にしたいのでoff
+    "lines-between-class-members": [
+      "error",
+      "always",
+      { exceptAfterSingleLine: true },
+    ],
+    "max-lines": [
+      "error",
+      {
+        max: 250,
+        skipBlankLines: true,
+        skipComments: true,
+      },
+    ],
+    "no-alert": "error",
+    "no-console": "error",
+    "no-promise-executor-return": "error",
+    // NOTE: import/no-relative-parent-importsだと@でのimportもNGになるのでno-restricted-importsで../を禁止する
+    "no-restricted-imports": ["error", { patterns: ["..*"] }],
+    "no-unused-expressions": "off", // duplidate @typescript-eslint/no-unused-expressions
+    "no-useless-constructor": "off",
+    "no-use-before-define": "off", // duplidate @typescript-eslint/no-use-before-define
+    "no-param-reassign": [
+      "error",
+      {
+        props: false,
+      },
+    ],
+
+    /**
+     * @typescript-eslint
+     */
+    "@typescript-eslint/consistent-type-assertions": [
+      "error",
+      // TODO: assertionStyle: never にしたい
+      { assertionStyle: "as", objectLiteralTypeAssertions: "never" },
```

Comment:
こちらもここではneverにしておいてproject側で上書きしようかと思います！
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-assertions.md#options

```suggestion
      { assertionStyle: "never" },
```